src/App.tsx

```diff
@@ -1,26 +1,34 @@
+import { Suspense, lazy } from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 
-import MainContainer from "./screens/MainContainer";
-import SignUp from "./screens/SignUp";
-import Login from "./screens/Login";
-import NotFound from "./screens/NotFound";
+const MainLayout = lazy(() => import("./screens/MainLayout"));
+const SignUp = lazy(() => import("./screens/SignUp"));
+const Login = lazy(() => import("./screens/Login"));
+const NotFound = lazy(() => import("./screens/NotFound"));
 
 function App() {
   return (
     <BrowserRouter>
-      <Routes>
-        {/* <Route element={<h1>Protect</h1>}> */}
-        <Route path="/" element={<MainContainer />}>
-          <Route index element={<h1>1</h1>} />
-          <Route path="first" element={<h1>2</h1>} />
-          <Route path="second" element={<h1>3</h1>} />
-        </Route>
-        {/* </Route> */}
+      <Suspense fallback={<h1>...Loading</h1>}>
+        <Routes>
+          {/* <Route element={<h1>Protect</h1>}> */}
+          <Route path="/" element={<MainLayout />}>
+            <Route index element={<h1>1</h1>} />
+            <Route path="/tickets" element={<h1>tickets</h1>} />
```

Comment:
Please create ROUTE const (for example in routes/index.ts or routes.ts) which will contains all path's. And you'll need to use ROUTE.TIKETS or ROUTER.HOME instead of 'tikets' and 'home' it'll help you to avoid typo issues and you can be sure that you use the same path in all places

Reply:
added routes.ts